### PR TITLE
Trials done

### PR DESCRIPTION
**Pull request by FrangoPious**
Check the trials.

### CODE REVIEW

File: 1stfile.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+1.git config --global user.name"username" (to linkup github(server) with local git( )
+2.git config -- global user.mail "mail"(to link with mail)
```
Comment (kulkarni90) [inline 1stfile.txt:2]: please check the command

File: 1stfile.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+3.git clone   -> To clone a repository into new directory.
+4.git init    -> Create an empty Git repository or reinitialize an existing one.
+5.git add     -> Add file contents to the index.
+6.git add.    -> Add all files contents to the index
+7.git branch  -> Lists branches present in repository.
+8.git branch -a -> Lists all branches in remote repository.
```
Comment (kulkarni90) [inline 1stfile.txt:8]: please do check the command usage/result.

File: thirdfile.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+hi all
```
Comment (kulkarni90) [inline thirdfile.txt:1]: please remove unnecessary files

**Review by kulkarni90 — COMMENTED** [review]
please do correct all the pointed commands or statements.

**Comment by VardhanRajeev** [comment]
Deleted the pull request

**Pull request closed without merging**